Migrate example pages to documentation

Sergiodero opened the pull request:
This update moves all examples (Schedule & Realtime) previously hosted in the Resource Library to the Technical Documentation section. It also updates the navigation to reflect these changes.

These changes might be bundled up with the Overview update from PR #16 

Do you might having a look at it @isabelle-dr ?

Sergiodero committed:
Migrate Realtime Overview content to main Overview

This change implements the following updates:

- Deletes Realtime Overview file
- Removes Realtime Overview from navigation in mkdocs.yml
- Updates Technical Documentation Overview to extend Schedule and realtime descriptions (incorporating summarized content from Realtime overview)

Sergiodero committed:
transfer examples to technical documentation

This update moves all examples (Schedule & Realtime) previously hosted in the Resource Library to the Technical Documentation section. It also updates the navigation to reflect these changes.

isabelle-dr reviewed (commented):
Thank you for having these in alphabetical order! It always bothered me on the current gtfs.org. 

I think we're missing the [Routes, stops, and trips](https://gtfs.org/schedule/examples/routes-stops-trips/) example, no?
Also, are we hosting the [Example GTFS feed](https://gtfs.org/schedule/example-feed/) somewhere else?

The changes to overview.md are indeed included there, but they represent the latest no it shouldn't cause problems.

Sergiodero committed:
Add routes, stops & trips example

This change adds the routes, stops & trips example and updates the navigation to reflect this change.

Sergiodero commented:
Thanks @isabelle-dr I've just added the Routes, Stops and Trips example in the latest commit. As for the [Example GTFS feed](https://gtfs.org/schedule/example-feed/), this one is currently hosted at the end of the getting started section, I think it makes sense to leave it there as a sample, any thoughts? 

@tzujenchanmbd do you mind taking a look as well? thank you both!

tzujenchanmbd commented on docs/documentation/schedule/examples/routes-stops-trips.md:
```diff
@@ -0,0 +1,140 @@
+
+- The `route_id` from [routes.txt](../../reference/#routestxt) that corresponds to MAX Orange is listed
+- The `service_id` from [calendar.txt](../../reference/#calendartxt) that corresponds to weekends is listed
+- Each records contains a unique ID for each trip
```
```suggestion
- Each records contains a unique ID for each trip.
```

tzujenchanmbd commented on docs/documentation/schedule/examples/routes-stops-trips.md:
```diff
@@ -0,0 +1,140 @@
+- The `shape_id` from [shapes.txt](../../reference/#shapestxt) that corresponds to the MAX Orange route towards Saddletowne is listed for the first record and the one for MAX Orange route towards Brentwood is listed for the second and third records
+
+
+The `shape_id=3030026` corresponds to the MAX Orange towards Saddletowne. The file below includes information on the points that outline the route as well as the distance between those points. With this information, it is possible to plot the route on a map for trip planning or analytics purposes.
```
> The file below includes information on the points that outline the route as well as the distance between those points.

- points that outline the "trip"
- "distance between those point" may confuse ppl? (`shape_dist_traveled` is for the distance from the first shape point to the point specified in this record)

Sergiodero replied:
Updated the text to: "The file below includes information on the points that outline the trip as well as the distance between each point and the start of the trip."

tzujenchanmbd commented on docs/documentation/schedule/examples/routes-stops-trips.md:
````diff
@@ -0,0 +1,140 @@
+
+```
+service_id,date,exception_type
+weekend_service,20220623,2
````
```suggestion
weekend_service,20220717,2
```

tzujenchanmbd reviewed (commented):
Please see the following suggestions, thanks!

Sergiodero committed:
Update docs/documentation/schedule/examples/routes-stops-trips.md

Fixes punctuation in line 86.

Co-authored-by: Tzu-Jen Chan <126435471+tzujenchanmbd@users.noreply.github.com>

Sergiodero committed:
Update docs/documentation/schedule/examples/routes-stops-trips.md

Fix date for service exceptions

Co-authored-by: Tzu-Jen Chan <126435471+tzujenchanmbd@users.noreply.github.com>

Sergiodero committed:
Update routes-stops-trips.md

Update text to clarify description of shapes.txt file

Sergiodero committed:
Merge branch 'migrate_example_pages_to_documentation_20240509' of https://github.com/MobilityData/elrond.gtfs.org into migrate_example_pages_to_documentation_20240509

Sergiodero commented:
Thanks @tzujenchanmbd! changes have been made to address your comments. Can any of you have a final look before approving the changes? Thank you both!

Sergiodero committed:
Fix grammar in example pages

Fix grammar in example pages

tzujenchanmbd reviewed (approved):
lgtm!
